integration_tests/test_integration.py

```diff
@@ -0,0 +1,124 @@
+            portal_root='https://dataservice.duke.edu',
+            openid_provider_id='123')
+
+        self.sender_username = 'api_user_sender'
+        self.sender_email = 'sender@d4s2.com'
+        self.sender_dds_id = 'dds_id_sender'
+        self.sender_password = 'secret1'
+        self.sender_user = django_user.objects.create_user(self.sender_username, password=self.sender_username,
+                                                           email=self.sender_email)
+        self.sender_email_template_set = EmailTemplateSet.objects.create(name='senderset')
+        self.sender_user_email_template_set = UserEmailTemplateSet.objects.create(
+            user=self.sender_user, email_template_set=self.sender_email_template_set)
+        self.delivery_sender_template = EmailTemplate.objects.create(
+            template_set=self.sender_email_template_set,
+            owner=self.sender_user,
+            template_type=EmailTemplateType.objects.get(name='delivery'),
+            subject='Sender Delivery Subject',
+            body='Sender Delivery Body')
+        self.accepted_sender_template = EmailTemplate.objects.create(
+            template_set=self.sender_email_template_set,
+            owner=self.sender_user,
+            template_type=EmailTemplateType.objects.get(name='accepted'),
+            subject='Sender Delivery Accepted Subject',
+            body='Sender Delivery Accepted Body')
+        DDSUserCredential.objects.create(endpoint=dds_endpoint, user=self.sender_user, token='123', dds_id='456')
+
+        self.recipient_username = 'api_user_recipient'
+        self.recipient_email = 'recipient@d4s2.com'
+        self.recipient_dds_id = 'dds_id_recipient'
+        self.recipient_password = 'secret2'
+        self.recipient_user = django_user.objects.create_user(self.recipient_username, password=self.recipient_password,
+                                                              email=self.recipient_email)
+        DDSUserCredential.objects.create(endpoint=dds_endpoint, user=self.recipient_user, token='789', dds_id='999')
+
+    def login_as_sender(self):
+        self.client.force_login(self.sender_user)
+
+    def login_as_recipient(self):
+        self.client.force_login(self.recipient_user)
+
+    def dds_user_fetch_one(self, ddsutil, user_id):
+        if user_id == self.sender_dds_id:
+            return Mock(full_name='Sender', email=self.sender_user.email)
+        if user_id == self.recipient_dds_id:
+            return Mock(full_name='Recipient', email=self.recipient_user.email)
+        raise ValueError("Invalid user id:" + user_id)
+
+    @patch('d4s2_api_v1.api.DDSUtil')
+    @patch('switchboard.dds_util.DDSUser')
+    @patch('switchboard.dds_util.DDSProjectTransfer')
+    @patch('switchboard.dds_util.RemoteStore')
+    @patch('d4s2_api.utils.Message')
+    def test_accepted_delivery_uses_senders_email_template_set(self, mock_message, mock_remote_store,
```

Comment:
Here is the integration test where a delivery sender creates and sends a delivery and then a recipient accepts it.